docs/guide/nested-module-subdirs.md

```diff
@@ -0,0 +1,113 @@
+# Nested module folders
+
+Apostrophe has optional support for nested subdirectories of modules, tucked
+inside `modules`.
+
+You must set the `nestedModuleSubdirs` option to `true` in `app.js`, like this:
+
+
+<AposCodeBlock>
+```javascript
+require('apostrophe')({
+  shortName: 'my-project',
+  nestedModuleSubdirs: true
+  // etc., you may have additional options here as always
+  modules: {
+    // You can still enable all modules here, or move some or all to modules.js
```

Comment:
Let's introduce `'products': {}` here as well. We can call that out and say we're including that as the start of the example. This is all we need to get the others once they're registered.

Reply:
I don't think I understand — there's no such feature. `modules.js` files are loaded automatically, otherwise the subdirs themselves have no significance other than convenience, that leaves no role for placing `modules: {}` here. Unless you're talking about `module: {}` (the piece type), which also doesn't make sense in isolation as near as I can tell.

Reply:
Okay, we've discussed this outside of this thread so I understand better.

The docs PR could definitely be clearer on the fact that `modules.js` is not necessary. In this example it seems there's no need for a modules.js file, so we should leave it out. If you want to include that feature we should include a clear case for when it would be used.